Update replication-subscribers-and-always-on-availability-groups-sql-… (#8695)
MicrosoftDocs/sql-docs: docs/database-engine/availability-groups/windows/replication-subscribers-and-always-on-availability-groups-sql-server.md

EXEC sp_addpullsubscription_agent 
        @publisher =  N'<publisher name>', 
        @subscriber = N'<availability group listener name>',

>         @Distributor = N'<Distributor availability group listener name>', --this parameter is not needed if Distribution DB is not in AG        

Reviewer: ```suggestion
        @Distributor = N'<Distributor availability group listener name>', -- this parameter should only be used if the Distribution database is part of an availability group. 
```

>         @Distributor = N'<Distributor availability group listener name>', -- this parameter should only be used if the Distribution database is part of an availability group. 

Reviewer 2: ```suggestion
        @distributor = N'<distributor availability group listener name>', -- this parameter should only be used if the distribution database is part of an availability group. 
```

Author: @rwestMSFT This looks good.

Reviewer 2: Please approve the changes - I don't have the ability to do so.
